src/main/java/frc/team2412/robot/commands/arm/SetArmCommand.java

```diff
@@ -31,6 +31,6 @@ public void end(boolean interrupted) {
 
 	@Override
 	public boolean isFinished() {
-		return true;
+		return armSubsystem.isArmNearGoal();
```

Comment:
Note that since isArmNearGoal() currently defines "near" as ~30 degrees, this command will end way before the arm is actually 'near' the goal as the driver likely defines it. This is enabling the `SetFullArmCommand` class to start `SetWristCommand` before the arm reaches its goal (which is good). But if `isArmNearGoal()` is tightened to better match what the driver (or autonomous) expects, that means the `SetFullArmCommand()` will wait longer to move the wrist, and take longer overall.